device-types/Intel/H2312XXLR2.yaml

```diff
@@ -0,0 +1,32 @@
+---
+manufacturer: Intel
+model: H2312XXLR2
+slug: intel-h2312xxlr2
+u_height: 2
+is_full_depth: true
+comments: |
+  [H2312XXLR2 Specifications on intel.com](https://www.intel.com/content/www/us/en/products/sku/87573/intel-server-chassis-h2312xxlr2/specifications.html)
+subdevice_role: parent
```

Comment:
Please add `weight` for the empty chassis if you have it

Reply:
Thank you for adding this, please remove the comment you inserted in the file near this area. In the future just respond within the change request :) After removing the comment you can resolve this conversation as well

Reply:
I removed the comment and will add the psu modules to this PR during the next hour. thank you very much for the help :)

Reply:
I have added the power supplies as module-types. I am not sure if i created them correctly but i will rework them asap if i know what to do :+1: